### PR TITLE
Fix npm version parsing in podspec and add release task.

### PR DESCRIPTION
**Pull request by alloy**
AUTOMATION

### CODE REVIEW

**Comment by alloy** [comment]
Ugh, started from wrong branch. Going to rectify that.

**Comment by orta** [comment]
Cool, was wondering about that

**Comment by alloy** [comment]
Did done.

File: scripts/release.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+sh('npm run bundle');
+sh('git add Pod/Assets && git commit -m "[Pod] Update JS bundle."');
+
+console.log('=> Creating version bump commit and tag.');
+sh('npm version ' + versionChange);
+sh('git push');
+sh('git push --tags');
+
+publishPodspec('Emission.podspec');
+publishPodspec('node_modules/react-native/React.podspec');
```
Comment (orta) [inline scripts/release.js]: havascript!

Comment (alloy) [inline scripts/release.js]: No shame.

**Comment by orta** [comment]
Yep, looks like it would work to me 👍

**Pull request merged**